### DISCUSSION TITLE
Getting disjoint geometries from a polygon

### DISCUSSION THREAD

**Discussion by lapankrz** [gama-platform/gama#91] [Q&A] [answered]
Hi, I have a question concerning the representation of polygons in GAMA. I have an agent, which shape is a union of a couple of polygons. Not all of them overlapped however, so the shape is comprised of a few disjoint polygons. After starting the experiment I checked its shape and it was a list of polygons, e.g [polygon(...), polygon(...),...]. I haven't been able to find a way to extract the individual polygons in GAML. I'd be grateful if someone told me if it's possible and how to do it. Thanks in advance!

**Comment by lesquoyb** [comment, marked as answer]
Hello,
You can access the list of geometries composing your shape via the **geometries** attribute. This information is currently missing from the doc but will be added soon. You can find the other attributes available for any geometry at [this url](https://gama-platform.org/wiki/DataTypes#geometry)

**Reply by lapankrz** [reply]
Thank you so much!
